### ISSUE TITLE
Please add a crates.io co-maintainer

### ISSUE THREAD

**Issue by ijackson** [dirs-dev/dirs-rs#53]
tl;dr: Please add a crates.io co-owner to improve the bus factor for this crate and dirs-ng.

Hi.

Relevantly, I'm the current maintainer of [`shellexpand`](https://lib.rs/crates/shellexpand), which I inherited from its author when they weren't able to work on it any more.  I was [recently asked](https://gitlab.com/ijackson/rust-shellexpand/-/merge_requests/12) to swtich from `dirs` to [`home`](https://lib.rs/crates/home).

The principal motivation seems to have been to avoid the MPL.  I [strongly disagreed](https://gitlab.com/ijackson/rust-shellexpand/-/merge_requests/12#note_1791092849); I did some evaluation of both crates and concluded that I would like to stay with `dirs`.

However, the biggest point against `dirs` was the single ownership of the crate name on crates.io.  I would respectfully request that you add a second crate owner.   Single crate ownership can cause serious inconvenience if the owner can't work on the crate for any reason.  (Indeed `shellexpand` [had to be forked for many months](https://github.com/netvl/shellexpand/issues/17) and `dirs` seems to have had a temporary fork too.)

Your co-owner doesn't really have to do anything, other than approve (or not!) volunteers who come forward if you (for any reason) aren't available.

Obviously you'll want to pick someone you feel is ideologically aligned, and has similar engineering taste, and so on.  Hopefully you'll have a friend or colleague you can prevail on.  For the record: I am volunteering, but I don't want this request to be read as a request for you to grant co-ownership to me.  I just want to avoid the risk iof having to rename the crate on crates.io; that would be seriously disruptive for your thousands of dependencies).

Thanks for your attention.

**Comment by soc** [comment]
Hi Ian,
thanks for letting me know – I have to think about how to best address this.
I'll report back when I have some results.
Thanks,
Simon

**Comment by soc** [comment]
Thank you for your well-articulated thoughts in https://gitlab.com/ijackson/rust-shellexpand/-/merge_requests/12#note_1791092849, by the way. :-)

Edit: Ah, I see the MR was created by this Utkarsh Gupta guy. He seems to be on some weird personal crusade "against" this library.
